pkg/cmd/server/admin/create_nodeconfig.go

```diff
@@ -193,11 +199,16 @@ func (o CreateNodeConfigOptions) Validate(args []string) error {
 	return nil
 }
 
-func CopyFile(src, dest string, permissions os.FileMode) error {
-	// copy the cert and key over
-	if content, err := ioutil.ReadFile(src); err != nil {
-		return err
-	} else if err := ioutil.WriteFile(dest, content, permissions); err != nil {
+func CopyFile(srcFiles []string, dest string, permissions os.FileMode) error {
```

Comment:
This looks like the wrong API.  Don't you need newlines between the file contents anyway?  That would make this even stranger.